docs/Compile/BuildWindows.md

```diff
@@ -139,57 +140,57 @@ Specify CMake parameters: -Dtigervnc_DIR=[TigerVNC installation path]/lib/cmake
         cmake .. -DCMAKE_INSTALL_PREIX=`pwd`/install
         cmake --build . --target install
 
-#### qtermwidget(Not currently supported on Windows)
-- Compile from sorce code
-  + Source code location:  [https://github.com/lxqt/qtermwidget](https://github.com/lxqt/qtermwidget)
+#### qtermwidget (Not currently supported on Windows)
+- Compile from source code
+  + Source-code location:  [https://github.com/lxqt/qtermwidget](https://github.com/lxqt/qtermwidget)
   + Specify CMake parameters: -Dqtermwidget5_DIR=[qtermwidget installation path]/lib/cmake/qtermwidget5
 
 #### libssh
 - Use vcpkg
-  + Source code location: https://github.com/microsoft/vcpkg/
+  + Source-code location: https://github.com/microsoft/vcpkg/
   + Specify CMake parameters: -DCMAKE_TOOLCHAIN_FILE=[vcpkg installation path]/scripts/buildsystems/vcpkg.cmake
 
         cd vcpkg
         vcpkg install libssh
 
-- Compile from sorce code
+- Compile from source code
   + Source code location:  [https://www.libssh.org](https://www.libssh.org)
   + Specify CMake parameters: -Dlibssh_DIR=[libssh installation path]/lib/cmake/libssh
 
 ### Compile this project
 - Project location: [https://github.com/KangLin/RabbitRemoteControl](https://github.com/KangLin/RabbitRemoteControl)
-- Download source code:
+- Download the source code:
 
       git clone https://github.com/KangLin/RabbitRemoteControl.git
 
 - CMake parameters:
-  + RabbitCommon_DIR: RabbitCommon source code location
-  + BUILD_FREERDP：If compile FreeRDP
-  + WinPR_DIR: [freerdp installation path]/lib/cmake/WinPR2
-  + FreeRDP_DIR: [freerdp installation path]/lib/cmake/FreeRDP2
-  + FreeRDP-Client_DIR: [freerdp installation path]/lib/cmake/FreeRDP-Client2
-  + tigervnc_DIR: [TigerVNC installation path]/lib/cmake
+  + RabbitCommon_DIR: RabbitCommon source-code location
+  + BUILD_FREERDP：If compiling FreeRDP
+  + WinPR_DIR: [FreeRDP installation path]/lib/cmake/WinPR2
+  + FreeRDP_DIR: [FreeRDP installation path]/lib/cmake/FreeRDP2
+  + FreeRDP-Client_DIR: [FreeFRP installation path]/lib/cmake/FreeRDP-Client2
+  + TigerVNC_DIR: [TigerVNC installation path]/lib/cmake
   + vncclient_DIR: [libvncserver installation path]/lib/cmake/LibVncServer
```

Comment:
Not sure "lib/cmake/LibVncServer" is correct

Reply:
I am will chanage to libVNCServer.